src/document-info.js

```diff
@@ -56,10 +54,13 @@ export function documentInfoForDoc(nodeOrDoc) {
           canonicalUrl = parseUrl(canonicalTag.href).href;
         }
         const pageViewId = getPageViewId(ampdoc.win);
-        const res = {sourceUrl, canonicalUrl, pageViewId};
-        Object.defineProperty(res, 'sourceUrl', {
-          get: () => getSourceUrl(ampdoc.getUrl()),
-        });
+        const res = {
+          get sourceUrl() {
```

Comment:
you want user to call data.getSourceUrl(), instead of data.sourceUrl? I think this `get` makes sure they can access sourceUrl just like before and that's what we want.